pages/tests/[testsByRunId].tsx

```diff
@@ -275,8 +307,17 @@ Tests.getInitialProps = async (context): Promise<Props> => {
   )
   const tests = await testsByTestRunIdReq.json()
 
+  const getTestRunById = await fetch(
+    `${process.env.deltaCore}/api/v1/test_run/${testsByRunId}`,
+    {
+      method: "GET",
+    }
+  )
+  const data = await getTestRunById.json()
+
```

Comment:
This called my attention a lot, in theory, all the data about the test_run should be already on the first endpoint `/api/v1/tests_history/test_run/<test_run_id>`... so I took a look into the Core Service, and I saw that I didn't added `test_run_data` into that's the endpoint, apologies for this, will fix the Core service now. so you could remove this second call and use the data from the only one endpoint 😄

Reply:
oh, true! Sorry for that, Cristian
yesss, so there is no need to change the props everywhere above